app/javascript/packs/simulator.js

```diff
@@ -0,0 +1,11 @@
+import $ from 'jquery';
+
+import 'jquery-ujs';
+
+import '../../../public/src/app';
+
+window.$ = $;
+window.jQuery = $;
```

Comment:
You can use this plugin instead of setting $ as global variable https://webpack.js.org/plugins/provide-plugin/#usage-jquery